packages/angular_devkit/build_angular/src/webpack/configs/common.ts

```diff
@@ -434,12 +438,26 @@ export async function getCommonConfig(wco: WebpackConfigOptions): Promise<Config
             name: 'vendor',
             chunks: (chunk) => chunk.name === 'main',
             enforce: true,
-            test: /[\\/]node_modules[\\/]/,
+            test: VENDORS_TEST,
           },
         },
       },
     },
-    plugins: [new NamedChunksPlugin(), new DedupeModuleResolvePlugin({ verbose }), ...extraPlugins],
+    plugins: [
+      new NamedChunksPlugin(),
+      new DedupeModuleResolvePlugin({ verbose }),
+      new DevToolsIgnorePlugin({
+        vendors: !vendorSourceMap &&
+          !!vendorChunk && {
```

Comment:
In case `vendorChunking` is not enabled, I think we should still enable this.  When vendorChunk is disabled vendors are still included, just in another bundle.

Reply:
Ok!

Reply:
Done.